scripts/get-resource-index.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const __basedir = path.join(__dirname, "..");
+const productionConfig = path.join(__basedir, 'env/production/config.json');
+
+const resourceIndexPrefix = process.env.RESOURCE_INDEX_PREFIX;
+
+function main() {
+  const config = JSON.parse(fs.readFileSync(productionConfig));
+  var resourceIndex = new URL(config['RESOURCE_INDEX']);
```

Comment:
This could be accessing `RESOURCE_INDEX` from our `config` module directly, e.g.

```js
import { RESOURCE_INDEX } from './src/config.js';
```

Reply:
Updated in https://github.com/nextstrain/nextstrain.org/pull/841/commits/538c63c3cb10797e463e6d56e8af91d0da146d34